<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1349</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My Contact Manger</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Find Contact</string>
        </property>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="geometry">
          <rect>
           <x>620</x>
           <y>10</y>
           <width>631</width>
           <height>661</height>
          </rect>
         </property>
         <property name="title">
          <string>Manage Contact</string>
         </property>
         <widget class="QGroupBox" name="gbCreateContact">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>251</width>
            <height>251</height>
           </rect>
          </property>
          <property name="title">
           <string>Create Contact</string>
          </property>
          <widget class="QLineEdit" name="txtNameCreate">
           <property name="geometry">
            <rect>
             <x>92</x>
             <y>30</y>
             <width>151</width>
             <height>26</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="lblName">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>63</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
          <widget class="QLabel" name="lblContact">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>70</y>
             <width>63</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Contact</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="txtContactCreate">
           <property name="geometry">
            <rect>
             <x>92</x>
             <y>70</y>
             <width>151</width>
             <height>26</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="lblCity">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>110</y>
             <width>63</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>City</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="txtCityCreate">
           <property name="geometry">
            <rect>
             <x>92</x>
             <y>110</y>
             <width>151</width>
             <height>26</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="lblCountry">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>150</y>
             <width>63</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Country</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="txtCountryCreate">
           <property name="geometry">
            <rect>
             <x>92</x>
             <y>150</y>
             <width>151</width>
             <height>26</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="btnSaveContactCreate">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>210</y>
             <width>93</width>
             <height>29</height>
            </rect>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
          <widget class="QLabel" name="lblinfo">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>190</y>
             <width>231</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </widget>
         <widget class="QGroupBox" name="gbUpdateContact">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>300</x>
            <y>90</y>
            <width>321</width>
            <height>301</height>
           </rect>
          </property>
          <property name="title">
           <string>Update Contact</string>
          </property>
          <widget class="QLineEdit" name="txtName_2">
           <property name="geometry">
            <rect>
             <x>92</x>
             <y>90</y>
             <width>151</width>
             <height>26</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="lblName_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>90</y>
             <width>63</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
          <widget class="QLabel" name="lblContact_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>130</y>
             <width>63</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Contact</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="txtContact_2">
           <property name="geometry">
            <rect>
             <x>92</x>
             <y>130</y>
             <width>151</width>
             <height>26</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="lblCity_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>170</y>
             <width>63</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>City</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="txtCity_2">
           <property name="geometry">
            <rect>
             <x>92</x>
             <y>170</y>
             <width>151</width>
             <height>26</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="lblCountry_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>210</y>
             <width>63</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Country</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="txtCountry_2">
           <property name="geometry">
            <rect>
             <x>92</x>
             <y>210</y>
             <width>151</width>
             <height>26</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="btnUpdateContact">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>270</y>
             <width>93</width>
             <height>29</height>
            </rect>
           </property>
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
          <widget class="QLabel" name="lblName_3">
           <property name="geometry">
            <rect>
             <x>8</x>
             <y>40</y>
             <width>63</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="txtName_3">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>40</y>
             <width>151</width>
             <height>26</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="btnFindForUpdate">
           <property name="geometry">
            <rect>
             <x>250</x>
             <y>40</y>
             <width>61</width>
             <height>29</height>
            </rect>
           </property>
           <property name="text">
            <string>Find</string>
           </property>
          </widget>
         </widget>
         <widget class="QGroupBox" name="gbDeleteContact">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>350</y>
            <width>321</width>
            <height>301</height>
           </rect>
          </property>
          <property name="title">
           <string>Delete Contact</string>
          </property>
          <widget class="QLineEdit" name="txtNameDelete">
           <property name="geometry">
            <rect>
             <x>92</x>
             <y>90</y>
             <width>151</width>
             <height>26</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="lblName_4">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>90</y>
             <width>63</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
          <widget class="QLabel" name="lblContact_3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>130</y>
             <width>63</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Contact</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="txtContactDelete">
           <property name="geometry">
            <rect>
             <x>92</x>
             <y>130</y>
             <width>151</width>
             <height>26</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="lblCity_3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>170</y>
             <width>63</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>City</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="txtCityDelete">
           <property name="geometry">
            <rect>
             <x>92</x>
             <y>170</y>
             <width>151</width>
             <height>26</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="lblCountry_3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>210</y>
             <width>63</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Country</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="txtCountry_3">
           <property name="geometry">
            <rect>
             <x>92</x>
             <y>210</y>
             <width>151</width>
             <height>26</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="btnDeleteDelete">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>270</y>
             <width>93</width>
             <height>29</height>
            </rect>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
          <widget class="QLabel" name="lblName_5">
           <property name="geometry">
            <rect>
             <x>8</x>
             <y>40</y>
             <width>63</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="txtDeleteByName">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>40</y>
             <width>151</width>
             <height>26</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="btnFindtoDelete">
           <property name="geometry">
            <rect>
             <x>250</x>
             <y>40</y>
             <width>61</width>
             <height>29</height>
            </rect>
           </property>
           <property name="text">
            <string>Find</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>295</width>
            <height>42</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="btnCreate">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string>Create</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnUpdate">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string>Update</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnDelete">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>551</width>
           <height>541</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QRadioButton" name="rdoName">
                <property name="text">
                 <string>Name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="rdoContact">
                <property name="text">
                 <string>Contact Number</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="rdoCity">
                <property name="text">
                 <string>City</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLineEdit" name="txtValue"/>
              </item>
              <item>
               <widget class="QPushButton" name="btnFind">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>Find</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="tblWidgetRecords"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1349</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
